test/db/cmd/cmd_px

```diff
@@ -108,16 +108,113 @@ EOF
 RUN
 
 NAME=pxr
-BROKEN=1
 FILE=bins/elf/crackme
-CMDS=pxr 0x30@0x00600ae0
+CMDS=<<EOF
+pxr 48 @ 0x00600ae0
+?e ===
+pxr 24 @ 0x00600ae0+32
+EOF
 EXPECT=<<EOF
 0x00600ae0 0x000000000040055e   ^.@..... @reloc.printf 4195678 (.plt) R X 'push 0'
 0x00600ae8 0x000000000040056e   n.@..... @reloc.puts 4195694 (.plt) R X 'push 1'
 0x00600af0 0x000000000040057e   ~.@..... @reloc.__libc_start_main 4195710 (.plt) R X 'push 2'
 0x00600af8 0x000000000040058e   ..@..... @reloc.fgets 4195726 (.plt) R X 'push 3'
 0x00600b00 0x000000000040059e   ..@..... @reloc.strcmp 4195742 (.plt) R X 'push 4'
 0x00600b08 0x00000000004005ae   ..@..... @reloc.fflush 4195758 (.plt) R X 'push 5'
+===
+0x00600b00 0x000000000040059e   ..@..... @reloc.strcmp 4195742 (.plt) R X 'push 4'
+0x00600b08 0x00000000004005ae   ..@..... @reloc.fflush 4195758 (.plt) R X 'push 5'
+0x00600b10 ..[ null bytes ]..   00000000 loc.__data_start
+EOF
+RUN
+
+NAME=pxr1248
+FILE=bins/elf/crackme
+CMDS=<<EOF
+pxr4 32@0x00600ae0
+pxr8 32@0x00600ae0
+pxr2 32@0x00600ae0
+e asm.bits=32
+pxr 32@0x00600ae0
+pxr2j 16@0x00600ae0~{}
```

Comment:
If I do `pxr2j` without setting `e asm.bits=32` I get wrong results. Why is that? I would expect to see 16bits-wide values.

Reply:
should be fixed now